### PR TITLE
#159274449 Setup unit test with coverage

### PR DESCRIPTION
**Pull request by megame24**
#### What does this PR do?
Setup unit test with code coverage
#### Description of Task to be completed?
- Setup unit test
- Integrate Codeclimate with the badge
- Integrate Coveralls with the badge
#### How should this be manually tested?
- Install the app (`npm install`)
- Run `npm test`
#### Any background context you want to provide?
- The express app had to be refactored out of the server(index.js) file to enable test runs exit without error
#### What are the relevant pivotal tracker stories?
#159274449
#### Screenshots (if appropriate)
screen shot 2018-07-26 at 8 12 32 pm
screen shot 2018-07-26 at 8 13 06 pm
#### Questions:
N/A

### CODE REVIEW

**Comment by megame24** [comment]
I will need to reconfigure the test to use Postgres, as it currently makes use of MongoDB.

File: index.js [hunk 1]
```diff
@@ -54,11 +56,10 @@ app.use(function(req, res, next) {
 // will print stacktrace
 if (!isProduction) {
     app.use(function(err, req, res, next) {
-        console.log(err.stack);
-
-        res.status(err.status || 500);
+        if(process.env.NODE_ENV !== 'test') console.log(err.stack);
```
Comment (rhenshaw56) [inline index.js:59]: fix indentation issues please

**Comment by rhenshaw56** [comment]
tend to the comments

File: index.js (hunk 1 above)
Comment (megame24) [reply index.js:59]: Alright, on it

**Pull request closed without merging**